Documentation/About/Changelog/4-1-0.rst

```diff
@@ -0,0 +1,22 @@
+.. include:: ../../Includes.txt
+
+==========================
+Version 4.1.0 - 2020/06/03
+==========================
+
+This release is a feature release. It contains the implementation of OAuth2 authorization protocol.
+
+Download
+========
+
+Download this version from the `TYPO3 extension repository <https://extensions.typo3.org/extension/mautic/>`__ or from
+`GitHub <https://github.com/mautic/mautic-typo3/releases/tag/v4.1.0>`__.
+
```

Comment:
Please add a dedicated "Added" section here and update the documentation on how to use the oAuth2 feature (link).